awx/job_template_launch_resource.go

```diff
@@ -76,18 +78,29 @@ func (r *JobTemplateLaunchGetResponse) JobTemplateData() *JobTemplate {
 type JobTemplateLaunchPostRequest struct {
 	Request
 
-	extraVars string
+	extraVars map[string]interface{}
 }
 
-func (r *JobTemplateLaunchPostRequest) ExtraVars(value string) *JobTemplateLaunchPostRequest {
+func (r *JobTemplateLaunchPostRequest) ExtraVars(value map[string]interface{}) *JobTemplateLaunchPostRequest {
 	r.extraVars = value
 	return r
 }
 
+func (r *JobTemplateLaunchPostRequest) ExtraVar(name string, value interface{}) *JobTemplateLaunchPostRequest {
+	r.extraVars[name] = value
```

Comment:
I think that here we need to check if `r.extraVars` is `nil`, and initialize it if it is, otherwise the first call to this method will panic:

```go
if r.extraVars == nil {
  r.extraVars = make(map[string]interface{})
}
r.extraVars[name] = value
```